executive/Q56477660/current/popolo-m17n.json

```diff
@@ -62,7 +62,7 @@
         }
       ],
       "associated_wikidata_positions": [
-        "Q132050"
+        "Q1467097"
       ],
```

Comment:
Is this expected behaviour to have Queensland (Q36074), and therefore the Governor of Queensland (Q1467097) pulled through as an area of Brisbane city council? Not sure it shouldn't be, just wasn't immediately obvious why it was appearing so frequently.

Reply:
Yes. Each area gets its parent pulled in, and their parents too, transitively, until you get to the top-level area.